district_data_import.py

```diff
@@ -144,6 +146,53 @@
             'ZIP': district_record['zip']
         })
 
+        # Update for potential switch of parent name order
+
+        curr_informal_sal = dp_donorrecord['INFORMAL_SAL']
+        curr_main_f_name = dp_donorrecord['FIRST_NAME']
+        curr_spouse_f_name = dp_donorrecord['SP_FNAME']
+
+        dp_donorrecord_with_pnames = district_data_utils.create_dp_donorrecord(district_record=district_record, school_year=args.school_year)
+        dp_donorrecord.update({
+            'FIRST_NAME': dp_donorrecord_with_pnames['FIRST_NAME'],
+            'LAST_NAME': dp_donorrecord_with_pnames['LAST_NAME'],
+            'SP_FNAME': dp_donorrecord_with_pnames['SP_FNAME'],
+            'SP_LNAME': dp_donorrecord_with_pnames['SP_LNAME'],
+            'EMAIL': dp_donorrecord_with_pnames['EMAIL'],
+            'SPOUSE_EMAIL': dp_donorrecord_with_pnames['SPOUSE_EMAIL'],
+            'MOBILE_PHONE': dp_donorrecord_with_pnames['MOBILE_PHONE'],
+            'SPOUSE_MOBILE': dp_donorrecord_with_pnames['SPOUSE_MOBILE'],
+            'SALUTATION': dp_donorrecord_with_pnames['SALUTATION'],
+            'OPT_LINE': dp_donorrecord_with_pnames['OPT_LINE']
```

Comment:
This doesn't work for `EMPLOYER`. Can we deal with it by adding a check to see if the spouse order is changing and if so, swizzle the `EMPLOYER` fields? (`EMPLOYER` and `SP_EMPLOYER` need to be added to the 271 report first.)

Reply:
To be clear I think the current logic is probably fine, I'm just asking how we'll deal with employer.

Reply:
This is really a late breaking requirement, and I think we all seem to be in agreement that this latest upload is ready to go. I would agree with Byung that we should offer this for the next upload.

Reply:
Right, I wasn't asking for a change here, as I tried to say. I was trying to provoke a discussion of how to deal with employer.